tests/core/MapFile.spec.ts

```diff
@@ -0,0 +1,52 @@
+import Compressor from '../../lib/core/versions/latest/util/Compressor';
+import ErrorCode from '../../lib/core/versions/latest/ErrorCode';
+import JasmineSidetreeErrorValidator from '../JasmineSidetreeErrorValidator';
+import MapFile from '../../lib/core/versions/latest/MapFile';
+import MapFileModel from '../../lib/core/versions/latest/models/MapFileModel';
+import SidetreeError from '../../lib/core/SidetreeError';
+
+describe('MapFile', async () => {
```

Comment:
Sanity: there's no positive test case ... shouldn't we add one?

Reply:
Discuss offline, `Observer` "E2E" test already covers this path, so will skip adding for this case.